string pose_name
---
geometry_msgs/PoseStamped pose_stamped
bool success
